src/trix/models/composition.coffee

```diff
@@ -470,7 +470,7 @@ class Trix.Composition extends Trix.BasicObject
       else
         range = [position, position + 1]
         position += 1
-    else if insertion.endPosition - 1 isnt 0
+    else if insertion.offset - 1 isnt 0
```

Comment:
It's a little confusing to have `insertion.index` and `insertion.offset` since they don't refer to start or end. Especially since we use `insertion.startPosition` earlier in this method. It looks like they're not actually used (except for this change) so what do you think about removing them and using `insertion.startLocation.offset` here instead?